src/Core/Animation/Skeleton.cpp

```diff
@@ -223,7 +225,7 @@ Vector3 Skeleton::projectOnBone( int boneIdx, const Ra::Core::Vector3& pos ) con
     CORE_ASSERT( length_sq != 0.f, "bone has lenght 0, cannot project." );
 
     // Project on the line segment
-    const Scalar t = Math::clamp( op.dot( dir ) / length_sq, (Scalar)0.0, (Scalar)1.0 );
+    const Scalar t = std::clamp( op.dot( dir ) / length_sq, (Scalar)0.0, (Scalar)1.0 );

```

Comment:
std::clamp or Ra::Core::saturate? (same every time we clamp between 0 and 1)

Reply:
well, i'm not a fan of saturate, so feel free to change it as you wish (but I would re recommend to remove saturate and replace by clamp ...)